src/coreclr/jit/emitarm64.cpp

```diff
@@ -15658,6 +15844,44 @@ void emitter::emitDispArrangement(insOpts opt)
             str = "8b";
             break;
         case INS_OPTS_16B:
+            str = "16b";
```

Comment:
any reason why these were added in `emitDispArrangement`?

Reply:
Those cases were already in `emitDispArrangement`, I think the diff just makes it look like they were added in. I moved all the SVE-specific cases to `emitDispSveArrangement`, and left the remaining cases in `emitDispArrangement` as-is.